### PR TITLE
Remove Chain ID

### LINKED ISSUE #137: Remove Chain IDs
### Issue To Be Solved
Remove Chain ID from configuration, and wherever it can be removed as it is not necessary in cases where we know the address as the Chain ID can be derived from the address.

### PR DESCRIPTION
**Pull request by sideninja**
Closes #137 

## Description
Remove ChainID from CLI and support old format.

### CODE REVIEW

File: pkg/flowcli/config/json/account.go [hunk 1]
```diff
@@ -25,32 +25,15 @@ import (
...
-
-	return flow.ChainID(rawChainID)
-}
-
 // transformAddress returns address based on address and chain id
-func transformAddress(address string, rawChainID string) flow.Address {
-	chainID := transformChainID(rawChainID, address)
-
+func transformAddress(address string) flow.Address {
+	// only allow service for emulator
```
Comment (psiemens) [inline pkg/flowcli/config/json/account.go:34]: 👍 nice, that makes sense to me

**Review by psiemens — APPROVED** [review]
Good riddance! 🧹

**Comment by psiemens** [comment]
@sideninja Is this a breaking change to the config format? If so, can you add the `Breaking Change` label to this PR?

**Comment by sideninja** [comment]
> @sideninja Is this a breaking change to the config format? If so, can you add the `Breaking Change` label to this PR?

It's backward compatible and non breaking but I will do further testing in QA.

**Pull request merged**